raml-headers-form.html

```diff
@@ -172,6 +177,27 @@
       @apply(--secondary-button-hover);
       @apply(--raml-headers-form-action-button-hover);
     }
+
+    .btn-icon {
+      display: none;
+      @apply(--raml-headers-form-action-button-icon);
+    }
+
+    .btn-icon:hover {
+      @apply(--raml-headers-form-action-button-icon-hover);
+    }
+
+    .delete-icon {
+      @apply(--raml-headers-form-action-button-delete);
+    }
+
+    .delete-icon:hover {
+      @apply(--raml-headers-form-action-button-delete-hover);
+    }
+
+    .anypoint-text-input {
+      @apply(--anypoint-text-input);
```

Comment:
This should be styled using `anypoint-text-container` element's styles: https://github.com/advanced-rest-client/anypoint-text-field/blob/stage/anypoint-text-container.html